Lib/Inc/CHumidifier.h

```diff
@@ -0,0 +1,27 @@
+/**
+ * @file CHumidifier.h
```

Comment:
Should probably get rid of this class. Does not look like we need it. Leave for now and we'll decide during pair programming session.